runtime/lib/component/turen.js

```diff
@@ -188,7 +188,7 @@ Turen.prototype.recoverPausedOnAwaken = function recoverPausedOnAwaken () {
   var currentAppId = this.runtime.life.getCurrentAppId()
 
   logger.info('unmute possibly paused bluetooth player')
-  this.bluetoothPlayer && this.bluetoothPlayer.resume()
+  this.bluetoothPlayer && this.bluetoothPlayer.unmute()
```

Comment:
BluetoothPlayer doesn't own the unmute function.

Reply:
The bluetoothPlayer is a2dp-adapter.

Reply:
See:
this.bluetoothPlayer = bluetooth.getAdapter(bluetooth.protocol.PROFILE.A2DP)
this bluetoothPlayer already use new bluetooth A2DP.

Reply:
Have renamed it: bluetoothPlayer --> bluetoothA2dp